### PR TITLE
[Resource]Add bulk volumes delete

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMPiVolumesDeleteBasic
--- PASS: TestAccIBMPiVolumesDeleteBasic (36.28s)
PASS
```

### CODE REVIEW

File: ibm/service/power/resource_ibm_pi_volumes_delete.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+	log.Printf("[DEBUG] Volumes delete accepted:  %s", volinfo.Summary)
+
+	for _, v := range d.Get(Arg_VolumeIDs).([]interface{}) {
+		_, err = isWaitForIBMPIVolumeDeleted(ctx, volClient, v.(string), d.Timeout(schema.TimeoutDelete))
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	}
+	d.SetId(fmt.Sprintf("%s/%s", cloudInstanceID, strings.Join(volumeIDs, "/")))
```
Comment (michaelkad) [inline ibm/service/power/resource_ibm_pi_volumes_delete.go:74]: @ismirlia Something similar to this or a new attribute with the list of the replicants. Which can then be used for the delete.

**Review by ismirlia — APPROVED** [review]

File: ibm/service/power/resource_ibm_pi_volumes_delete.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+func ResourceIBMPIVolumesDelete() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMPiVolumesDeleteCreate,
+		ReadContext:   resourceIBMPiVolumesDeleteRead,
+		DeleteContext: resourceIBMPiVolumesDeleteDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			Arg_CloudInstanceID: {
+				Description: "Cloud Instance ID of a PCloud Instance.",
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_volumes_delete.go:29]: Description here doesn't match the description in the markdown below.

File: website/docs/r/pi_volumes_delete.html.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Timeouts
+
+pi_volumes_attach provides the following [Timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) configuration options:
+
+* `delete` - (Default 30 minutes) Used for deleting volumes in power virtual server.
```
Comment (michaelkad) [inline website/docs/r/pi_volumes_delete.html.markdown:42]:
```suggestion
* **delete** - (Default 30 minutes) Used for deleting volumes.
```

**Comment by ismirlia** [comment]
Opened: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5672 closing this one

**Pull request closed without merging**